mava/utils/loggers/base.py

```diff
@@ -173,13 +186,20 @@ def make_logger(
         return logger
 
     def _path(self, subdir: Optional[str] = None) -> str:
-        if subdir:
-            path = str(self._directory / self._time_stamp / subdir / self._label)
+        if isinstance(self._directory, str):
+            if subdir:
+                path = f"{self._directory}/{self._time_stamp}/{subdir}/{ self._label}"
+            else:
+                path = f"{self._directory}/{self._time_stamp}/{self._label}"
```

Comment:
Maybe would be cleaner to make a list of this stuff, ignore the empty/None items and then `"/".join(path)`

Reply:
I agree that would be cleaner, but I prefer the readability of how it looks now. Do you think the cleaner implementation is worth the potential readability hit?

Reply:
Ye I thought it might be the same. So can leave as is